packages/eslint-config-fbjs/package.json

```diff
@@ -19,12 +19,12 @@
     "test": "echo \"Error: no test specified\" && exit 1"
   },
   "peerDependencies": {
-    "babel-eslint": "^7.2.3 || ^8.0.0 || ^9.0.0",
+    "babel-eslint": "^7.2.3 || ^8.0.0 || ^9.0.0 || ^10.0.0",
```

Comment:
11 is in beta, but it requires a babel config file. not sure if you wanna handle that somehow?

Reply:
Do you need the 11 for Jest, or would the 10 be enough for now?

Reply:
It's needed for TypeScript, but we'll probably go for https://github.com/typescript-eslint/typescript-eslint/ and not `babel-eslint`